test/test.js

```diff
@@ -49,6 +49,15 @@ $.mockjax( {
 } );
 
 $.mockjax( {
+	url: "callback.php",
+	response: function( settings ) {
+		this.responseText = settings.data.responseText();
+		this.responseStatus = settings.data.responseStatus || 200;
+		this.responseTime = settings.data.responseTime || 100;
+	}
+} );
```

Comment:
I see no need in registering another `mockjax` handler, there is already a similar one with url `response.php`, please adapt your test to use it instead.